src/renderer/screens/exchange/Swap2/History/index.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import HistoryV1 from "../../swap/History";
+
+/*
+ ** As History screen doesn't have a specific v2 yet, this component
+ ** only import and return v1 of the history screen.
+ ** For the moment, UI is broken and need to be fixed. I wait to discuss
+ ** with the design team before fixing it.
+ */
+const History = () => <HistoryV1 />;
```

Comment:
Since we are forking away from v1, don't import anything from v1 but rather copy it and we will modify it inside the v2. As far as the UI being broken, it should at least take the full width/height of the component and render the placeholder, not this small centered version

Reply:
Got it.